toolboxTreppe/toolboxTreppe.ino

```diff
@@ -106,9 +103,10 @@ void animation1(uint8_t& y, uint8_t& x, uint8_t& gegenstrecke) {
 }
 //----------------------------- Debug Funktion ----------------
 
-void debugPrint(const char* output[]) {
+void debugPrint(const char* output[], ...) {
+  va_list args;
   if (digitalRead(buttonPin1) && digitalRead(buttonPin2)) {
-    Serial.println(output);
+    Serial.println(output, args);
```

Comment:
and here its wrong.
Serial.println takes 2 params: 1) the string 2) a "mode" like print in hex
you need to call sprintf() to get the final string and pass it to serial.println

Reply:
I know already saw that in the error messages on thrusday but had no time to change it yet